diff -uNr lxc-5.0.3/src/lxc/tools/lxc_copy.c lxc-5.0.3.mod/src/lxc/tools/lxc_copy.c
--- lxc-5.0.3/src/lxc/tools/lxc_copy.c	2023-07-26 06:00:11.000000000 +0800
+++ lxc-5.0.3.mod/src/lxc/tools/lxc_copy.c	2024-02-08 18:54:53.707153154 +0800
@@ -677,6 +677,49 @@
 	return -1;
 }
 
+int
+getsubopt (char **optionp, char *const *tokens, char **valuep)
+{
+  char *endp, *vstart;
+  int cnt;
+
+  if (**optionp == '\0')
+    return -1;
+
+  /* Find end of next token.  */
+  endp = strchrnul (*optionp, ',');
+
+  /* Find start of value.  */
+  vstart = memchr (*optionp, '=', endp - *optionp);
+  if (vstart == NULL)
+    vstart = endp;
+
+  /* Try to match the characters between *OPTIONP and VSTART against
+     one of the TOKENS.  */
+  for (cnt = 0; tokens[cnt] != NULL; ++cnt)
+    if (strncmp (*optionp, tokens[cnt], vstart - *optionp) == 0
+    && tokens[cnt][vstart - *optionp] == '\0')
+      {
+    /* We found the current option in TOKENS.  */
+    *valuep = vstart != endp ? vstart + 1 : NULL;
+
+    if (*endp != '\0')
+      *endp++ = '\0';
+    *optionp = endp;
+
+    return cnt;
+      }
+
+  /* The current suboption does not match any option.  */
+  *valuep = *optionp;
+
+  if (*endp != '\0')
+    *endp++ = '\0';
+  *optionp = endp;
+
+  return -1;
+}
+
 static int parse_mntsubopts(char *subopts, char *const *keys, char *mntparameters)
 {
 	while (*subopts != '\0') {
